### ISSUE TITLE
Uncaught TypeError: deprecationSite.getFileName is not a function

### ISSUE THREAD

**Issue by luggage66** [jumpkick/graphviz-preview#18]
I updated to atom 1.2.1 and now I get this error below in any .dot file, while attempting to open the preview. 

**Atom Version**: 1.2.1
**System**: Mac OS X 10.11.1
**Thrown From**: [graphviz-preview](https://github.com/jumpkick/graphviz-preview) package, v1.0.0

### Stack Trace

Uncaught TypeError: deprecationSite.getFileName is not a function

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/grim/lib/grim.js:56

TypeError: deprecationSite.getFileName is not a function
  at global.__grim__.deprecate (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/grim/lib/grim.js:56:36)
  at Promise.done (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:1025:5)
  at Object.module.exports.toggle (/Users/dmull/.atom/packages/graphviz-preview/lib/graphviz-preview.coffee:44:68)
  at atom-text-editor.subscriptions.add.atom.commands.add.graphviz-preview:toggle (/Users/dmull/.atom/packages/graphviz-preview/lib/graphviz-preview.coffee:12:93)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:160:19)
  at AtomEnvironment.module.exports.AtomEnvironment.dispatchApplicationMenuCommand (/Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:986:28)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:169:7)

```

### Commands

```
  9x -9:23.9.0 core:undo (atom-text-editor.editor.is-focused)
     -9:19 tabs:close-all-tabs (li.tab.sortable.active.right-clicked)
  2x -8:58.9.0 graphviz-preview:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -7:51.2.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
     -0:02.8.0 graphviz-preview:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "excludeVcsIgnoredPaths": false
  }
}
```

### Installed Packages

```coffee
# User
git-log, v0.4.1
graphviz-preview, v1.0.0
language-awk, v0.1.6
language-dot, v1.6.0
language-glsl, v2.0.1
language-graphql, v0.2.0
language-jade, v0.6.2
language-latex, v0.6.1
linter-eslint, v5.2.0
nuclide-flow, v0.0.32
react, v0.12.10

# Dev
No dev packages
```

**Comment by luggage66** [comment]
The above error happens after updating atom to 1.2.1.

**Comment by deftdawg** [comment]
Try with the just released 1.7.0
